.runs/issue-56-git-fallback/build/impl_changes_summary.md

```
@@ -0,0 +1,113 @@
+    }
+
+    // Try git fallback
+    if let Some(content) = try_git_show(file) {
+        return Some((content, DiscoveryMethod::GitShow));
+    }
+
+    None
+}
+```
+
+### Test Output
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)